template-patches/python-configuration.patch

```diff
@@ -1,16 +1,15 @@
 diff --git a/python/configuration.mustache b/python/configuration.mustache
-index dceab63..b2d56ab 100644
+index ac51800..2511102 100644
 --- a/python/configuration.mustache
 +++ b/python/configuration.mustache
-@@ -259,7 +259,7 @@ conf = {{{packageName}}}.Configuration(
+@@ -265,7 +265,7 @@ conf = {{{packageName}}}.Configuration(
          """Adding retries to override urllib3 default value 3
          """
-         # Disable client side validation
+         # Enable client side validation
 -        self.client_side_validation = True
 +        self.client_side_validation = False
  
-     def __deepcopy__(self, memo):
-         cls = self.__class__
+         self.socket_options = None
+         """Options to pass down to the underlying urllib3 socket
```

Comment:
This is going to replace lines 12 and 13 in the old template, can they really be removed?

Also, I remember that clients generated with the new version of openapi-generator already had client side validation disabled so maybe this template is not necessary anymore?

Reply:
No, the patch isn't replacing the lines you highlighted. Maybe the combination of GitHub diffs and patch diffs is preventing you from reading clearly, if you pull the source code and check you'll see that those lines aren't removed.

Reply:
Yes, right, I forgot this is a diff.

As for the second point: do we still need to apply this patch?

Reply:
Yes, unfortunately client-side validation it's still active in the 7.0.0 https://github.com/OpenAPITools/openapi-generator/blob/c37fa8a4e8399ebd76d7f190136fd4b851c9aa7e/modules/openapi-generator/src/main/resources/python/configuration.mustache#L268